css/css-flexbox/reference/flex-aspect-ratio-img-row-014-expected.xht

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+ <head>
+  <title>CSS Reftest Reference</title>
+  <link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com" />
+  <style type="text/css"><![CDATA[
+  span
+  {
+  display: inline-block;
```

Comment:
Nit: Really weird format/indentation here.

Reply:
Again, using similar style as other `flex-aspect-ratio-img-*`tests. I'll change that.